server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -113,11 +114,12 @@ public ObjectMapper build(BuilderContext context) {
         protected ObjectMapper createMapper(String name,
                                             int position,
                                             long columnOID,
+                                            boolean isDropped,
                                             String fullPath,
                                             Dynamic dynamic,
                                             Map<String, Mapper> mappers,
                                             @Nullable Settings settings) {
-            return new ObjectMapper(name, position, columnOID, fullPath, dynamic, mappers, settings);
+            return new ObjectMapper(name, position, columnOID, isDropped, fullPath, dynamic, mappers, settings);
         }
     }
 
```

Comment:
I think we are missing parser changes, smth like https://github.com/crate/crate/commit/1ca05298a4efbfeb5df1077bfdaaf126d6382cea#diff-224944376a5c852dcccb4b9e7b1f663b902c6f0e52e54b2863bb0f59558fc851R147-R149